CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+- Failed deploys to shinyapps.io will now output build logs. Posit Cloud application deploys will also output build logs once supported server-side.
```

Comment:
maybe reword to "once server-side is supported"?

Reply:
I'm referring to connect-based outputs here, not server-side rendering.

Reply:
Oh, I misunderstood the comment.  The words `server-side` are automatically associated to static outputs in my mind.